src/workspace-hack/Cargo.toml

```diff
@@ -135,7 +135,7 @@ chrono = { version = "0.4.25", default-features = false, features = ["alloc", "c
 clap = { version = "3.2.24", features = ["derive", "env", "wrap_help"] }
 console = { version = "0.15.5", default-features = false, features = ["ansi-parsing", "unicode-width"] }
 criterion = { version = "0.4.0", features = ["async_tokio", "html_reports"] }
-crossbeam-deque = { version = "0.8.2" }
+crossbeam-deque = { version = "0.8.3" }
```

Comment:
What happened here?

Reply:
0.8.3 introduced a new API to grab a batch of things from the injector, which we really like to have for lgalloc.